porter invoke does check the error when retrieving the claim

When executing `porter invoke` it's possible for a panic to occur

```
(base) Ubuntu:~$ porter invoke --action uninstallAll my-app --tag myimage:mytag --param cluster-name=my-cluster --param target=azure --cred my-app --debug
invoking custom action uninstallAll on my-app...
Resolved storage plugin to storage.porter.filesystem
/home/user/.porter/porter plugin run storage.porter.filesystem
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x16a10f4]
goroutine 1 [running]:
get.porter.sh/porter/pkg/cnab/provider.(*Runtime).Invoke(0xc00024e7e0, 0x7fff448ae584, 0xc, 0x7fff448ae591, 0x9, 0xc000194960, 0x4b, 0xc00059eb40, 0xc00059eb10, 0xc0005a40e0, ...)
        /home/vsts/work/1/s/gopath/src/get.porter.sh/porter/pkg/cnab/provider/invoke.go:60 +0xb4
get.porter.sh/porter/pkg/porter.(*Porter).InvokeBundle(0xc000240820, 0x7fff448ae584, 0xc, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7fff448ae591, ...)
        /home/vsts/work/1/s/gopath/src/get.porter.sh/porter/pkg/porter/invoke.go:60 +0x4dc
main.buildBundleInvokeCommand.func2(0xc000298c80, 0xc0001152c0, 0x1, 0xc, 0x0, 0x0)
        /home/vsts/work/1/s/gopath/src/get.porter.sh/porter/cmd/porter/bundle.go:181 +0x9a
github.com/spf13/cobra.(*Command).execute(0xc000298c80, 0xc000115200, 0xc, 0xc, 0xc000298c80, 0xc000115200)
        /home/vsts/work/1/s/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003f4780, 0x0, 0x18ed7c0, 0xc0000ac058)
        /home/vsts/work/1/s/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
        /home/vsts/work/1/s/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
        /home/vsts/work/1/s/gopath/src/get.porter.sh/porter/cmd/porter/main.go:18 +0x27
```

It's because on this line:

https://github.com/deislabs/porter/blob/9110c71d4d65c5b1d2d0705b618b035efd93c672/pkg/cnab/provider/invoke.go#L55

we don't check if an error occurred when retrieving the claim, for example when there isn't a claim because the bundle was uninstalled.

### Porter Version

```
(base) Ubuntu:~$ porter version
porter v0.24.0-beta.1 (6da074e)
```

--- a/build/azure-pipelines.pr-automatic.yml
+++ b/build/azure-pipelines.pr-automatic.yml
@@ -6,6 +6,7 @@
 trigger: none
 pr: 
   - master
+  - release-*
 
 pool:
   vmImage: 'Ubuntu 16.04'

--- a/build/azure-pipelines.pr-manual.yml
+++ b/build/azure-pipelines.pr-manual.yml
@@ -6,6 +6,7 @@
 trigger: none
 pr: 
   - master
+  - release-*
 
 pool:
   vmImage: 'Ubuntu 16.04'

--- a/pkg/cnab/provider/invoke.go
+++ b/pkg/cnab/provider/invoke.go
@@ -27,7 +27,7 @@ func (d *Runtime) getClaim(bun *bundle.Bundle, actionName, claimName string) (*c
 				}
 			}
 		}
-		return nil, false, errors.Wrapf(err, "could not load claim %s", claimName)
+		return nil, false, errors.Wrapf(err, "could not load bundle instance %s", claimName)
 	}
 	return &c, false, nil
 }
@@ -53,6 +53,9 @@ func (d *Runtime) Invoke(action string, args ActionArguments) error {
 	}
 
 	c, isTemp, err := d.getClaim(bun, action, args.Claim)
+	if err != nil {
+		return err
+	}
 
 	// Here we need to check this again
 	// If provided, we should set the bundle on the claim accordingly

--- a/pkg/cnab/provider/invoke_test.go
+++ b/pkg/cnab/provider/invoke_test.go
@@ -189,21 +189,32 @@ func Test_ClaimLoading(t *testing.T) {
 			want: result{
 				claim: eClaim,
 				temp:  false,
-				err:   errors.Wrap(claim.ErrClaimNotFound, "could not load claim nonexist"),
+				err:   errors.Wrap(claim.ErrClaimNotFound, "could not load bundle instance nonexist"),
 			},
 		},
 	}
 
 	for _, tc := range tests {
-		in := tc.in
-		want := tc.want
-		_, temp, err := d.getClaim(in.bun, in.action, in.claim)
-		assert.Equalf(t, want.temp, temp, "%s: expected temp=want.temp", tc.name)
-		if want.err == nil {
-			assert.NoErrorf(t, err, "%s: expected no error", tc.name)
-		} else {
-			assert.Errorf(t, err, "%s: expected error", tc.name)
-			assert.EqualErrorf(t, want.err, err.Error(), "%s: expected error %s, got %s", tc.name, want.err, err)
-		}
+		t.Run(tc.name, func(t *testing.T) {
+			in := tc.in
+			want := tc.want
+			_, temp, err := d.getClaim(in.bun, in.action, in.claim)
+			assert.Equalf(t, want.temp, temp, "getClaim returned an unexpected temporary flag")
+			if want.err == nil {
+				assert.NoErrorf(t, err, "getClaim failed")
+			} else {
+				assert.EqualErrorf(t, err, want.err.Error(), "getClaim returned an unexpected error")
+			}
+		})
 	}
+}
+
+func TestInvoke_NoClaimBubblesUpError(t *testing.T) {
+	r := NewTestRuntime(t)
+
+	args := ActionArguments{
+		Claim: "mybuns",
+	}
+	err := r.Invoke("custom-action", args)
+	require.EqualError(t, err, "could not load bundle instance mybuns: Claim does not exist")
 }

